common/tasks/rosa-hcp/generate-cluster-name.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: generate-cluster-name
+spec:
+  results:
+    - name: cluster-name
+      description: Openshift cluster name
+  steps:
+    - name: generate-cluster-name
+      image: registry.access.redhat.com/ubi9/openssl:9.1-2
+      script: |
+        #!/usr/bin/env bash
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        
+        CLUSTER_NAME="kx-$( openssl rand -hex 5 )"
```

Comment:
@xinredhat  What do you think about using the pipelinerun name to get a string for the cluster name. Then we will be able to easily map which cluster comes from which run.

Reply:
@ascerra pipelinerun name unfortunately is to long for rosa naming. I think needs 15 chars and pr run have more

Reply:
Right I was thinking we could just cut the output to use only the last 5 characters

Reply:
makes sense for me. Feel free to open a pr